File: Test_01/Task01.java
package Test;
import java.util.Scanner;

public class Task01 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Запрос ввода от пользователя
        System.out.println("Введите количество секунд: ");
        int seconds = scanner.nextInt();

        // Форматирование секунд
        String formattedSeconds = formatSeconds(seconds);

        // Вывод отформатированных секунд
        System.out.println(formattedSeconds);
    }

    public static String formatSeconds(int seconds) {
        int days = seconds / 86400;
        seconds %= 86400;
        int hours = seconds / 3600;
        seconds %= 3600;
        int minutes = seconds / 60;
        seconds %= 60;

        return days + " days " + hours + " hours " + minutes + " minutes " + seconds + " seconds";
    }
}

File: Test_01/Task01.py
#Процедура:
def format_seconds(seconds):
    days = seconds // 86400
    seconds -= days * 86400
    hours = seconds // 3600
    seconds -= hours * 3600
    minutes = seconds // 60
    seconds -= minutes * 60
    formatted = f'{days} days {hours} hours {minutes} minutes {seconds} seconds '
    return formatted

#Функция:
def format_seconds(seconds):
    days = seconds // 86400
    seconds -= days * 86400
    hours = seconds // 3600
    seconds -= hours * 3600
    minutes = seconds // 60
    seconds -= minutes * 60
    return f'{days} days {hours} hours {minutes} minutes {seconds} seconds '

seconds = int(input("Введите кол-во секунд: "))
formatted_seconds = format_seconds(seconds)
print(formatted_seconds)  # Output: '1 days 10 hours 17 minutes 36 seconds '